Gemfile

```diff
@@ -11,7 +11,7 @@ gem 'sqlite3', '~> 1.4'
 gem 'puma', '~> 4.1'
 # Use SCSS for stylesheets
 gem 'sass-rails', '>= 6'
-# Transpile app-like JavaScript. Read more: https://github.com/rails/webpacker
+# Transpile app-like JavaScript. Read |more: https://github.com/rails/webpacker
```

Comment:
Please remove this typo. You may need to check the changes on each file using the `git diff` command before committing your changes.